rebar.lock

```diff
@@ -18,11 +18,11 @@
   0},
  {<<"aeserialization">>,
   {git,"https://github.com/aeternity/aeserialization.git",
-      {ref,"ecde108939ca407d8f92e991f062e51fc13f3beb"}},
+      {ref,"816bf994ffb5cee218c3f22dc5fea296c9e0882e"}},
   0},
  {<<"aesophia">>,
   {git,"https://github.com/aeternity/aesophia.git",
-      {ref,"3271d6fba49fd03499b6b93f18130ff822a9f8e3"}},
+      {ref,"fd6dac22573a36cd01b1db3bcef79ca7c819a662"}},
```

Comment:
I do not see https://github.com/aeternity/aesophia/commit/fd6dac22573a36cd01b1db3bcef79ca7c819a662 merged to master of its repo so this commit may disappear so I do not see this rebar.lock as mergeable to master.

Reply:
Fixed now.